CHANGELOG.md

```diff
@@ -4,6 +4,83 @@ IMPROVEMENTS:
 * Use `consul-k8s` subcommand to perform `tls-init` job. This allows for server certificates to get rotated on subsequent runs.
   Consul servers have to be restarted in order for them to update their server certificates [[GH-749](https://github.com/hashicorp/consul-helm/pull/721)]
 
+BREAKING CHANGES:
+* `connectInject.centralConfig.defaultProtocol` is no longer supported. Instead,
+   [`controller.enabled`](https://www.consul.io/docs/k8s/helm#v-controller-enabled) must be set to true and
+   a [`ServiceDefaults`](https://www.consul.io/docs/agent/config-entries/service-defaults) resource
+   must be created to set *each* service's protocol, e.g.:
+   
+   ```yaml
+   apiVersion: consul.hashicorp.com/v1alpha1
+   kind: ServiceDefaults
+   metadata:
+     name: my-service-name
+   spec:
+     protocol: "http"
+   ```
+  
+  This setting is being removed because it didn't support changing the
+  protocol after a service was first run and because it didn't work in secondary
+  datacenters.
+  
+  See [Upgrade to CRDs](https://www.consul.io/docs/k8s/crds/upgrade-to-crds)
+  for more information on how to migrate to using custom resources to manage
+  service protocols. [[GH-763](https://github.com/hashicorp/consul-helm/pull/763)] 
+* `connectInject.centralConfig.proxyDefaults` is no longer supported. Instead,
+   [`controller.enabled`](https://www.consul.io/docs/k8s/helm#v-controller-enabled) must be set to true and
+   a [`ProxyDefaults`](https://www.consul.io/docs/agent/config-entries/proxy-defaults) resource
+   must be created.
+  
+  This setting is being removed because it didn't support any modifications to the
+  config after the cluster was first installed.
+  
+  If you were previously setting this, see [Upgrade to CRDs](https://www.consul.io/docs/k8s/crds/upgrade-to-crds)
+  for more information on how to upgrade. [[GH-763](https://github.com/hashicorp/consul-helm/pull/763)] 
+* `connectInject.centralConfig.enabled` is no longer supported. Instead,
+   central config will always be set to `true`. If you were previously setting
+   this to `true` then this change has no effect. If you were previously setting
+   this to `false`, then you must instead, override this setting in
+   `client.extraConfig` and `server.extraConfig`:
+   
+   ```yaml
+   client:
+     extraConfig: |
+       {"enable_central_service_config": false}
+   server:
+     extraConfig: |
+       {"enable_central_service_config": false}
+   ``` 
+  [[GH-763](https://github.com/hashicorp/consul-helm/pull/763)] 
+* The `consul.hashicorp.com/connect-service-protocol` annotation on Connect pods is
+  no longer supported with the latest version of `consul-k8s`.
```

Comment:
Should we update this to specify a version number or say will not be supported in versions of consul-k8s beyond x?
`latest` feels generic in this context.